spec/models/social_networking/serializers/shared_item_serializer_spec.rb

```diff
@@ -4,50 +4,68 @@ module SocialNetworking
   module Serializers
     RSpec.describe SharedItemSerializer do
       let(:participant) { instance_double(Participant, is_admin: true) }
-      let(:thought) { double("thought", participant_id: 1) }
-      let(:shared_item) do
-        instance_double(
-          SharedItem,
-          item: thought,
-          item_label: "label",
-          id: 1,
-          created_at: Time.zone.now,
-          is_public: true,
-          action_type: "",
-          likes: [],
-          comments: []
-        )
-      end
-      let(:serialized_shared_item) do
-        SharedItemSerializer
-          .new(shared_item)
-          .to_serialized
-      end
 
       describe ".to_serialized" do
-        before do
-          allow(thought).to receive(:participant) { participant }
-          allow(thought).to receive(:to_serialized)
-          allow(thought).to receive(:description) { "" }
+        context "a valid shared item type" do
+          let(:thought) { double("thought", participant_id: 1) }
```

Comment:
Social networking does not have access to the Thought class, so I don't think this is possible.